CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 Changes to Calva.
 
 ## [Unreleased]
+- [Begin work on enabling strictNullChecks in the TypeScript config.](https://github.com/BetterThanTomorrow/calva/pull/1568)
```

Comment:
```suggestion
- Maintenance: [Begin work on enabling strictNullChecks in the TypeScript config.](https://github.com/BetterThanTomorrow/calva/pull/1568)
```

As a way to communicate that the users should not expect any new behaviour.

Reply:
Oh, great idea, that'll be helpful ☺️